data_files/server11.key

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
It looks like you've regenerated `server11.key`, was this intentional?

Reply:
NP ;)
`server11.key` should be a new file, not replacing an existing one. Looking at the diff of this PR it seems that this should be correct.